app/assets/translations/appConstants/bebbo/uk.ts

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Should we change "Search in Bebbo" to "Search" in this file?

Reply:
Please ignore 'search' string for now as we are not using it.